### PR TITLE
Add Beginners PR guide to CONTRIBUTING.org

### PR DESCRIPTION
**Pull request by robbyoconnor**

### CODE REVIEW

File: CONTRIBUTING.org [hunk 1]
```diff
@@ -81,6 +81,7 @@ Submit your contribution against the =develop= branch. You should not use
 your =master= branch to modify Spacemacs, this branch is considered to be
 read-only.
 
+You may want to [[https://github.com/syl20bnr/spacemacs/wiki/Beginner%2527s-Guide-to-Contributing-a-Pull-Request-to-Spacemacs][read our beginner guide for Pull Requests]].
```
Comment (StreakyCobra) [inline CONTRIBUTING.org]: Without a new line before `PR=Pull Request` they are in the same paragraph/line. Is this in purpose?

Comment (robbyoconnor) [inline CONTRIBUTING.org]: It really doesn't matter -- it's just pedantic

Comment (robbyoconnor) [inline CONTRIBUTING.org]: Added the a line between them.

**Comment by StreakyCobra** [comment]
:+1:

File: CONTRIBUTING.org (hunk 1 above)
Comment (StreakyCobra) [inline CONTRIBUTING.org]: BTW you really like weird percent-symbols, like `%2527`? :joy: i.e. the link is broken.

File: CONTRIBUTING.org [hunk 2]
```diff
@@ -81,6 +81,8 @@ Submit your contribution against the =develop= branch. You should not use
 your =master= branch to modify Spacemacs, this branch is considered to be
 read-only.
 
+You may want to [[https://github.com/syl20bnr/spacemacs/wiki/Beginner%2527s-Guide-to-Contributing-a-Pull-Request-to-Spacemacs][read our beginner’s guide for Pull Requests]].
```
Comment (StreakyCobra) [inline CONTRIBUTING.org]: BTW you really like weird percent-symbols, like `%2527`? :joy: i.e. the link is broken.

Comment (robbyoconnor) [inline CONTRIBUTING.org]: URL Encoding dude: https://github.com/syl20bnr/spacemacs/wiki/Beginner%27s-Guide-to-Contributing-a-Pull-Request-to-Spacemacs

Comment (robbyoconnor) [inline CONTRIBUTING.org]: URL encoding -- that `25` was a mistake, the rest was not.

**Comment by StreakyCobra** [comment]
Fix #3898

**Comment by robbyoconnor** [comment]
:+1:

**Comment by balajisivaraman** [comment]
Nice! :+1:

**Comment by StreakyCobra** [comment]
Thank you :+1: Cherry-picked into the `develop` branch. You can safely delete your branch :smiley:

I removed the trailing white-space in 67f515a2c0e2c1fa5195e33e0936f42637480320

**Pull request closed without merging**